static/docs/commands-reference/status.md

```diff
@@ -50,21 +50,45 @@ detected, `dvc status` prints this message:
 ```
 
 This says that no differences were detected, and therefore that no stages would
-be run again if `dvc repro` were executed.
+be run again by `dvc repro`.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
-differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+DVC-file (stage) with differences, the changes in _dependencies_ and/or
+_outputs_ that differ are listed. For each item listed, either the file name or
+the checksum is shown, and additionally a status word is shown describing the
+changes in checksum or status. Given list provides a reference to both, the
+status of the DVC-file and differences in the DVC-files:
 
 - For the local workspace:
-  - _changed_ means the file has changed
+
+  - _changed checksum_ means actual checksum of the files present in workspace
+    doesn't match the one saved in DVC-files;
```

Comment:
it deserves a better explanation tbh. Also, `files` -> `the file`, `in DVC-files` -> `in the DVC-file` ... no need to mention workspace here. The rule of thumb - would you understand anything from this description if you were reading this first time?

Reply:
How about "means that the update checksum of the data doesn't match the one saved in the DVC-files" as I suggested in https://github.com/iterative/dvc.org/pull/416#pullrequestreview-257759178?

Reply:
updated it in my latest commit, kindly review them.